tests/servers/test_basehttp.py

```diff
@@ -29,13 +29,13 @@ def test_https(self):
 
         handler = WSGIRequestHandler(request, '192.168.0.2', None)
 
-        with captured_stderr() as stderr:
+        with patch_logger('django.request.runserver', 'info') as messages:
```

Comment:
@fcurella Hi :) do you think you can merge fcurella#1 so we can re-run the tests?